bungle/lua-resty-jq
# adds tests and reworks filter config options

- tests(*) add busted tests and a Makefile
- ensure context is set to nil on teardown
- use local table.concat
- treat join as raw, just without new lines
- check context is initialised before using
- remove raw_no_lf option (this is join)
- test result of parse against kind, rather than nil
- remove error flag; compile returns error directly
- make error messages more uniform
- remove dump and flags options, add compact_output
- add ascii_output option
- add sort_keys option
